src/app/calendar/_components/CalendarReservationForm.tsx

```diff
@@ -96,19 +96,27 @@ export function CalendarReservationForm({
               type="time"
               list="time"
               required
-              defaultValue={new Date(editedReservation?.startTime ?? "07:00")
-                .toTimeString()
-                .slice(0, 5)}
+              defaultValue={
+                editedReservation
+                  ? new Date(editedReservation?.startTime)
```

Comment:
if you do a ternary here to check if editedReservation is not undefined you can omit `?`
```typescript
? new Date(editedReservation.startTime)
```